src/stage/mouse-observer.ts

```diff
@@ -456,13 +456,13 @@ class MouseObserver {
 
   _setCanvasPosition (event: any) {  // TODO
     const box = this.domElement.getBoundingClientRect()
-    let offsetX, offsetY
-    if ('offsetX' in event && 'offsetY' in event) {
+    let offsetX, offsetY;
+    if ('clientX' in event && 'clientY' in event) {
+      offsetX = event.clientX - left
```

Comment:
`left` and `top` are undefined variables. `box.left` and `box.right` are probably the expected values.

Reply:
Whoops, you're right! I was copied from an out of date fork. I'll get that fixed